tests/braille-specs/sw-ke-g1-2.yaml

```diff
@@ -0,0 +1,73 @@
+# Swahili (Kenya) Braille Grade 1.2
+# Copyright (C) 2022 by Compass Braille <http://www.compassbraille.org>
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
```

Comment:
Does it make sense to combine all the tests into a single file? I think this helps in getting the overview and understanding the similarities/differences between the various grades. It also allows to remove some duplication (by applying multiple tables to the same test data).

Reply:
Is there an example .yaml file where multiple tables are applied to the same test data? I can't quite see how to do this.

Reply:
Ah I think I have found one:
litdigits6dots_backward.yaml
?

Reply:
Indeed

Reply:
Just done this